test.py

```diff
@@ -0,0 +1,6 @@
+import requests
+import pprint
+
+r = requests.get("http://lookup-service-prod.mlb.com/json/named.player_info.bam?sport_code='mlb'&player_id='493316'")
+
+pprint.pprint(r.json())
```

Comment:
one letter variables are bad practice. use res instead

Reply:
Got it. Made this change.